pkg/kubelet/dockershim/network/kubenet/kubenet_linux.go

```diff
@@ -569,9 +717,7 @@ func (plugin *kubenetNetworkPlugin) addContainerToNetwork(config *libcni.Network
 	klog.V(3).Infof("Adding %s/%s to '%s' with CNI '%s' plugin and runtime: %+v", namespace, name, config.Network.Name, config.Network.Type, rt)
 	// The network plugin can take up to 3 seconds to execute,
```

Comment:
This comment is no longer correct in this context, since the locks no longer exist here, also it would seem moving the locking logic into AddNetwork will make this problematic